code/game/objects/items/weapons/batons.dm

```diff
@@ -38,7 +42,7 @@
 	if(HAS_TRAIT(user, TRAIT_CLUMSY) && prob(50))
 		user.visible_message("<span class='danger'>[user] accidentally clubs [user.p_them()]self with [src]!</span>", \
 							 "<span class='userdanger'>You accidentally club yourself with [src]!</span>")
-		user.Weaken(stun_time)
+		//user.KnockDown(knockdown_duration)
```

Comment:
I assume you want to either change this or remove this. If you want to remove it, then don't forget to remove the commented out code